cloudinit/distros/netbsd.py

```diff
@@ -13,7 +13,7 @@
 LOG = logging.getLogger(__name__)
 
 
-class Distro(cloudinit.distros.bsd.BSD):
+class NetBSD(cloudinit.distros.bsd.BSD):
```

Comment:
Given that the naming doesn't indicate this is used for OpenBSD too, I'd like to see a comment/docstring here that indicates that it is.